### PR TITLE
BZ-1877344: Added pod phases and statuses

### PR DESCRIPTION
**Pull request by sagidlow**
Applies to 4.5+
BZ Link: https://bugzilla.redhat.com/show_bug.cgi?id=1877344
QE approval needed: @skordas 
Preview Link: https://deploy-preview-31920--osdocs.netlify.app/openshift-enterprise/latest/applications/application_life_cycle_management/odc-viewing-application-composition-using-topology-view.html#odc-interacting-with-applications-and-components_viewing-application-composition-using-topology-view

### CODE REVIEW

File: modules/odc-interacting-with-applications-and-components.adoc [hunk 1]
```diff
@@ -16,8 +16,16 @@ This feature is available only when you create applications using the *From Git*
...
-* The status or phase of the pod is indicated by different colors and tooltips as *Running* (image:odc_pod_running.png[title="Pod Running"]), *Not Ready* (image:odc_pod_not_ready.png[title="Pod Not Ready"]), *Warning*(image:odc_pod_warning.png[title="Pod Warning"]), *Failed*(image:odc_pod_failed.png[title="Pod Failed"]), *Pending*(image:odc_pod_pending.png[title="Pod Pending"]), *Succeeded*(image:odc_pod_succeeded.png[title="Pod Succeeded"]), *Terminating*(image:odc_pod_terminating.png[title="Pod Terminating"]), or *Unknown*(image:odc_pod_unknown.png[title="Pod Unknown"]).
-For more information about pod status, see the link:https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase[Kubernetes documentation].
+* The status or phase of the pod is indicated by different colors and tooltips as:
```
Comment (mburke5678) [inline modules/odc-interacting-with-applications-and-components.adoc:19]: I see this is existing text. But should _or phase_ be set-off in commas? This makes it seem like _status_ and _phase_ are different things. (I assume they are the same thing.)

File: modules/odc-interacting-with-applications-and-components.adoc [hunk 2]
```diff
@@ -16,8 +16,16 @@ This feature is available only when you create applications using the *From Git*
...
+** *Running* (image:odc_pod_running.png[title="Pod Running"]): The Pod has been bound to a node, and all of the containers have been created. At least one container is still running, or is in the process of starting or restarting.
```
Comment (mburke5678) [inline modules/odc-interacting-with-applications-and-components.adoc:20]: s/Pod/pod

Do we need the comma after node?

File: modules/odc-interacting-with-applications-and-components.adoc (hunk 1 above)
Comment (sagidlow) [reply modules/odc-interacting-with-applications-and-components.adoc:19]: They are the same thing. However, previously we didn't provide full descriptions of each phase. We only linked to the Kubernetes doc. There were 3 phases not documented as they are on the OCP side. So, a bulleted list made more sense.

File: modules/odc-interacting-with-applications-and-components.adoc (hunk 2 above)
Comment (sagidlow) [reply modules/odc-interacting-with-applications-and-components.adoc:20]: I can remove :)

File: modules/odc-interacting-with-applications-and-components.adoc [hunk 3]
```diff
@@ -16,8 +16,16 @@ This feature is available only when you create applications using the *From Git*
...
+** *Not Ready* (image:odc_pod_not_ready.png[title="Pod Not Ready"]): In pods that are running multiple containers, not all containers are ready. 
+** *Warning*(image:odc_pod_warning.png[title="Pod Warning"]): Containers in pods must be terminated, however termination did not succeed. Some containers may be other states.
```
Comment (mburke5678) [inline modules/odc-interacting-with-applications-and-components.adoc:22]: _Containers in pods must be terminated_
Does this mean the pods are being terminated, or someone/something needs to?

File: modules/odc-interacting-with-applications-and-components.adoc [hunk 4]
```diff
@@ -16,8 +16,16 @@ This feature is available only when you create applications using the *From Git*
...
+** *Failed*(image:odc_pod_failed.png[title="Pod Failed"]): All containers in the Pod have terminated, and at least one container has terminated in failure. That is, the container either exited with non-zero status or was terminated by the system.
```
Comment (mburke5678) [inline modules/odc-interacting-with-applications-and-components.adoc:23]: s/Pod/pod

File: modules/odc-interacting-with-applications-and-components.adoc [hunk 5]
```diff
@@ -16,8 +16,16 @@ This feature is available only when you create applications using the *From Git*
...
+** *Pending*(image:odc_pod_pending.png[title="Pod Pending"]): The Pod has been accepted by the Kubernetes cluster, but one or more of the containers has not been set up and made ready to run. This includes time a Pod spends waiting to be scheduled as well as the time spent downloading container images over the network.
+** *Succeeded*(image:odc_pod_succeeded.png[title="Pod Succeeded"]): All containers in the Pod have terminated in success, and will not be restarted.
+** *Terminating*(image:odc_pod_terminating.png[title="Pod Terminating"]): When a Pod is being deleted, it is shown as *Terminating* by some kubectl commands. *Terminating* status is not one of the Pod phases. A Pod is granted a term to terminate gracefully, which defaults to 30 seconds.
```
Comment (mburke5678) [inline modules/odc-interacting-with-applications-and-components.adoc:26]: s/term/period
We use the phrase _graceful termination period_ [elsewhere](https://docs.openshift.com/container-platform/4.7/nodes/pods/nodes-pods-priority.html#priority-preemption-graceful_nodes-pods-priority).

**Comment by mburke5678** [comment]
@sagidlow A few nits. Otherwise LGTM

**Pull request merged**